data/libs/Equipment.lua

```diff
@@ -294,6 +296,9 @@ HyperdriveType.HyperjumpTo = function (self, ship, destination)
 	if not fuel_use then
 		return "INSUFFICIENT_FUEL"
 	end
+	if wheels == 1 then
```

Comment:
technically, this should be != 0, otherwise you can initiate a jump while the gears are moving (while moving the gears float goes through the values from 0 to 1).